esp/platform/espcontext.cpp

```diff
@@ -123,7 +123,7 @@ class CEspContext : public CInterface, implements IEspContext
         if (m_txSummary)
         {
             m_txSummary->tailor(this);
-            m_txSummary->log(getTxSummaryLevel(), getTxSummaryGroup(), getTxSummaryStyle());
+            m_txSummary->log(getTxSummaryLevel(), getTxSummaryGroup(), getTxSummaryStyle() | TXSUMMARY_FWD_OTEL);
```

Comment:
we might want to avoid forwarding to otel (really jtrace) if tracing not enabled.